.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Tue May 11 17:01:00 BST 2021
+gradle.version=6.7
```

Comment:
Thought gradle was related to Java, why do we need this in the fact-admin? :) 

Think there are also .gradle files too. Cached folders